app/merchant/views.py

```diff
@@ -81,9 +83,23 @@ def order_items():
 @login_required
 @merchant_required
 def manage_cart():
+    # used to show/hide the order modal
+    confirm_order = request.args.get('confirm_order', default=False, type=bool)
+    # vendor_id to order from. if None, order from all
+    vendor_id = request.args.get('vendor_id', type=int)
+    if vendor_id not in CartItem.get_vendor_ids():
+        vendor = None
+    else:
+        vendor = Vendor.query.get(vendor_id)
+
     vendor_items_dict = CartItem.get_vendor_cart_items_dict()
-    return render_template('merchant/manage_cart.html',
-                           vendor_items_dict=vendor_items_dict)
+    return render_template(
+        'merchant/manage_cart.html',
+        vendor_items_dict=vendor_items_dict,
+        confirm_order=confirm_order,
+        vendor=vendor,
+        get_total_price=CartItem.get_total_price
+    )
 
 
 @merchant.route('/items/<int:listing_id>')
```

Comment:
Yep, I worked on it yesterday to fix the admin page bug, and also to give admin's access to that view. After this PR is merged let's get that one in as well